init.go

```diff
@@ -51,7 +54,7 @@ func init() {
 	}
 
 	config = &oauth2.Config{
-		RedirectURL:  "https://" + HostURL + "/home",
+		RedirectURL:  "http://" + HostURL + "/",
```

Comment:
Also, keep the secure protocol `https`
```suggestion
		RedirectURL:  "https://" + HostURL + "/",
```

Reply:
Good catch! I forgot to switch it back after tinkering with it for localhost
Adressed in 9588515